### ISSUE TITLE
Invalid count value

### ISSUE THREAD

**Issue by MaximeEvangeliste** [dstoyanoff/directus-auto-migrate#5]
Getting this error when running the initial command: `npx directus-auto-migrate my-first-migration`

```
RangeError: Invalid count value
    at String.repeat (<anonymous>)
    at Object.message (/Users/max/Desktop/directus-test/node_modules/directus-auto-migrate/src/utils/logger.ts:32:26)
    at Object.<anonymous> (/Users/max/Desktop/directus-test/node_modules/directus-auto-migrate/src/migrate.ts:19:9)
    at step (/Users/max/Desktop/directus-test/node_modules/directus-auto-migrate/dist/migrate.js:33:23)
    at Object.next (/Users/max/Desktop/directus-test/node_modules/directus-auto-migrate/dist/migrate.js:14:53)
    at fulfilled (/Users/max/Desktop/directus-test/node_modules/directus-auto-migrate/dist/migrate.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)

```

**Comment by dstoyanoff** [comment]
Hey @MaximeEvangeliste,
Can you check what version of the script are you using? I've fixed that once and wondering if you have an older version or it's a different issue.

Would also appreciate if you share version of node, what shell and os you are using.

 🍻

**Comment by vomchik** [comment]
got the same error, looks like something is wrong here https://github.com/dstoyanoff/directus-auto-migrate/blob/main/src/utils/logger.ts#L32

it happens when I run the script first time

after removing this line, everything works fine https://github.com/dstoyanoff/directus-auto-migrate/blob/main/src/migrate.ts#L19

**Comment by dstoyanoff** [comment]
Hello @vomchik, thanks for your confirmation and yeah, I understand where the code breaks. What I am trying to understand is why it does break, so I will appreciate if you can as well give some additional information about your environment and the version of the package that's getting resolved for you.

**Comment by vomchik** [comment]
> Hello @vomchik, thanks for your confirmation and yeah, I understand where the code breaks. What I am trying to understand is why it does break, so I will appreciate if you can as well give some additional information about your environment and the version of the package that's getting resolved for you.

node: v14.16.0
ts-node: v10.2.1
directus-auto-migrate: v0.3.1
directus: v9.0.0-rc.92

**Comment by dstoyanoff** [comment]
Fixed in 0.3.2

**Comment by vomchik** [comment]
thanks a lot 🎉
